### ISSUE TITLE
SA1629 reports false-positives for <example> Tags

### ISSUE THREAD

**Issue by DanielHabenicht** [DotNetAnalyzers/StyleCopAnalyzers#2914]
For the given example: 

```csharp
/// <example>true</example>
public bool Active { get; set; }
```

the following Warning is thrown on compile: 
>Warning SA1629 Documentation text should end with a period	Cool.Project C:\Path\to\git\Cool.Project\src\Cool.Project\Models\AppDto.cs	37 Active

In my opinion this is a false-positive. How do you think?

**Comment by sharwell** [comment]
The `<example>` element is nearly identical to the `<remarks>` element for tools that render documentation. The sample code presented above would result in a paragraph of text 'true' with no context or explanation.

> ## Summary
>
> This is the summary.
>
> ## Remarks
>
> Here are some remarks.
>
> ## Example
>
> true

**Comment by DanielHabenicht** [comment]
Well in my case I am using the `<example>` tag in order to generate example data with [swashbuckle](https://github.com/domaindrivendev/Swashbuckle).

**Comment by sharwell** [comment]
You can exclude `example` from the list of elements covered by this rule in stylecop.json. See [Configuration.md](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/Configuration.md#text-ending-with-a-period) for details.

**Comment by DanielHabenicht** [comment]
But isn't it failing the purpose of providing untouched example data?

**Comment by DanielHabenicht** [comment]
Yeah ok, got it. It's just a use case for rest API endpoints. Thanks for your time!

**Comment by daniel-white** [comment]
It would be nice if this was included by default

**Comment by sharwell** [comment]
@daniel-white `example` is checked by default for the same reason `remarks` is checked by default. I'm not a big fan of SA1629 overall, but it wouldn't make sense to exclude this tag for users who enable the diagnostic.
